### ISSUE TITLE
Error while running event BlueprintExtensions::on_configuration_changed: Invalid surface name

### ISSUE THREAD

**Issue by eduran84** [dewiniaid/BlueprintExtensions#10]
I get the following error when loading a map. This happens directly after activating Blueprint Extensions for the first time.

```
 385.472 Script @__BlueprintExtensions__/modules/landfill.lua:43: Computing collision box overrides...
 385.599 Error AppManagerStates.cpp:1397: The mod Blueprint Extensions caused a non-recoverable error.
Please report this error to the mod author.

Error while running event BlueprintExtensions::on_configuration_changed
Invalid surface name: Surface names must not be blank and must be unique.
stack traceback:
	__BlueprintExtensions__/modules/landfill.lua:46: in function '?'
	__BlueprintExtensions__/control.lua:60: in function 'call_module_methods'
	__BlueprintExtensions__/control.lua:96: in function <__BlueprintExtensions__/control.lua:93>
stack traceback:
	[C]: in function 'create_surface'
	__BlueprintExtensions__/modules/landfill.lua:46: in function '?'
	__BlueprintExtensions__/control.lua:60: in function 'call_module_methods'
	__BlueprintExtensions__/control.lua:96: in function <__BlueprintExtensions__/control.lua:93>
```
[factorio-current.log](https://github.com/dewiniaid/BlueprintExtensions/files/3040302/factorio-current.log)

 Let me know in case you need any additional info.

**Comment by dewiniaid** [comment]
Sorry for the delay, Github opted to stop notifying me of issues.

I'm investigating this now.

**Comment by dewiniaid** [comment]
Should be fixed in 0.4.4, now available.
